### PR TITLE
Docker testnet + repo restructure.

### PR DESCRIPTION
**Pull request by sebohe**
Resolves: https://github.com/hashcloak/Meson/issues/23 by doing the following:
- Moved the Meson code into `plugin`
- Added Dockerfile for `plugin`
- New ops dir that contains all the config files for the non voting testnet.
- Removed `./client` because a new repo was created at: https://github.com/hashcloak/Meson-client

New features:
- `Makefile` that builds all the necessary containers with:
    - `make all`
   - `make up-nonvoting` to start a testnet.
  - Uses both rinkeby and goerli testnets.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -55,10 +55,13 @@ Ticker and RPC connection information, currency.toml:
 ::
```
Comment (Mikerah) [inline README.rst]: This `README.rst` should be in the `plugin` folder. Create a new `README` for the entire repo with basic instructions on how to run everything.

File: cmd/proxy/proxy.go [hunk 2]
```diff
@@ -111,8 +111,8 @@ type Currency struct {
 }
 
 // GetParameters : Returns params from Currency struct
-func (k *Currency) GetParameters() (map[string]string) {
-	return k.params
+func (k *Currency) GetParameters() (map[string]string, error) {
```
Comment (Mikerah) [inline cmd/proxy/proxy.go]: This plugin has to satisfy the `ServicePlugin` interface defined in https://github.com/katzenpost/server/blob/master/cborplugin/client.go#L63-L79. 

Keep this method as it is.

**Comment by sebohe** [comment]
> Another thing I would like to bring up is the committing of the `.pem` files for authorities, providers and mixes.
> 
> As this is for this basic testnet, we don't need to worry to much about this in order to make it reproducible. However, for subsequent testnets, there should be instructions on how to generate these `.pem` files and they should not be committed to a public repository.

Yes. This is something I was thinking of including in this issue: https://github.com/hashcloak/Meson/issues/27

File: cmd/proxy/proxy.go (hunk 2 above)
Comment (sebohe) [reply cmd/proxy/proxy.go]: I removed the changes but removed the error var waiting for the function call: https://github.com/hashcloak/Meson/pull/28/files#diff-3002d695552ec7e7ebbaaf04a21d3439L70-L74

**Comment by Mikerah** [comment]
Why was this merged?
I didn't try running the instructions yet.
Also, you didn't move the old `README.rst` to the `plugin` folder.
The structure of the `plugin folder` should be as follows:
`plugin`
   - `README.rst`
   - `cmd/meson`: contains `main.go`
   -  `pkg/meson`: contains everything else.

**Comment by sebohe** [comment]
Sorry, this wasn't supposed to be merged yet.

About the plugin folder. I misunderstood what we talked about earlier in the chat.

**Pull request merged**